HACKING.md

```diff
@@ -217,6 +217,20 @@ $ aws s3 mb my-fcos-bucket
 
 And provide it to `--bucket` below.
 
+### [OPTIONAL] Slack integration
+
+If you want to be able to have build status messages appear in Slack,
+create a `slack-api-token` secret:
+
+```
+$ echo -n "$TOKEN" > slack-token
+$ oc create secret generic slack-api-token --from-file=token=slack-token
+```
+
+You can obtain a token (Slack calls this the "Integration Token
+Credential ID") when creating a new instance of the Jenkins CI app in
+your Slack workspace.
```

Comment:
is that a "Legacy token" as described here: https://api.slack.com/docs/token-types ?

Reply:
Ahh thanks. No it's not. It's not a token associated with a user scope. It's a bot token generated by the Jenkins Slack app.

Reply:
Yeah that wasn't obvious to me. I went to the slack web ui and tried to find a place to create a token. I think this extra bit of text helps the "gap". If there is a link to "how to create the app" that would help even more.